frontend/app/src/components/accounts/manual-balances/ManualBalanceTable.vue

```diff
@@ -237,6 +230,20 @@ const headers = computed(() => [
     width: '50'
   }
 ]);
+
+const { show } = useConfirmStore();
+
+const showDeleteConfirmation = (id: number) => {
+  set(pendingDeletion, id);
+
+  show(
+    {
+      title: tc('manual_balances_table.delete_dialog.title'),
+      message: tc('manual_balances_table.delete_dialog.message')
+    },
+    deleteBalance
```

Comment:
The `pendingDeletion` part we can possible remove, if instead of passing the `deleteBalance` reference we pass a `() => deleteBalance(id)`